### DISCUSSION TITLE
version

### DISCUSSION THREAD

**Discussion by lilicapucinepineau** [mlco2/codecarbon#723] [Q&A]
Hello, 
I am using Code Carbon at work, and I am trying to automatically retrieve the version that is used when codecarbon is called. Is there any way for me to do that ? I haven't found a way yet from the codecarbon options.
Thank you.

**Comment by benoit-cty** [comment]
Hello, you could do it like that :

```py
>>> import codecarbon
>>> codecarbon.__version__
'2.8.1'
```
